src/uu/touch/src/touch.rs

```diff
@@ -84,13 +84,9 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     ) {
         (Some(reference), Some(date)) => {
             let (atime, mtime) = stat(Path::new(reference), !matches.get_flag(options::NO_DEREF))?;
-            if let Ok(offset) = humantime_to_duration::from_str(date) {
-                let mut seconds = offset.whole_seconds();
-                let mut nanos = offset.subsec_nanoseconds();
-                if nanos < 0 {
-                    nanos += 1_000_000_000;
-                    seconds -= 1;
-                }
+            if let Ok(offset) = parse_datetime::from_str(date) {
```

Comment:
please move this change into a different PR

Reply:
I can, but the point was to get "green" and stop all the failed CI builds.

Reply:
It doesn't really matter IMHO